### PR TITLE
Bug 1032371 - Display users who have completed task on task detail page

### PR DESCRIPTION
**Pull request by tessie**

### CODE REVIEW

File: oneanddone/users/tests/test_views.py [hunk 1]
```diff
@@ -103,3 +103,12 @@ def test_get_object_non_existent_username(self):
 
         with self.assertRaises(Http404):
             self.view.get_object()
+
+    def test_get_object_existing_userid(self):
```
Comment (bobsilverberg) [inline oneanddone/users/tests/test_views.py]: Please move this test above `test_get_object_non_existent_username` (to keep them in alphabetical order).

**Comment by bobsilverberg** [comment]
Great work @tessie! This is ready to go after you fix the one comment I added above about rearranging the tests to put them in alphabetical order. Ping me again once that's done.

**Comment by tessie** [comment]
@bobsilverberg Thanks for the review :).have made the changes .if it is according to alphabetical order should not  def test_get_object_existing_userid(self) come before def test_get_object_existing_username(self)

**Comment by bobsilverberg** [comment]
@tessie Yes, that looks good. This is ready to merge. Can you please squash the commits and then I will merge. Thanks!

**Comment by tessie** [comment]
@bobsilverberg squashed.Thank you

**Comment by bobsilverberg** [comment]
Great, thanks @tessie!

**Pull request merged**